src/src/WebServer/Metrics.cpp

```diff
@@ -7,12 +7,13 @@
 # include "../../_Plugin_Helper.h"
 # include "../Helpers/ESPEasyStatistics.h"
 # include "../Static/WebStaticData.h"
-HELPERS_ESPEASY_MATH_H
+//HELPERS_ESPEASY_MATH_H  //clumsy-stefan: what's this for?
```

Comment:
No idea why it ended up in the code.
You can remove the entire line.

Reply:
Done.